hugo.toml

```diff
@@ -199,6 +199,10 @@ no = 'Oh snap! Sorry to hear that. Please <a href="https://github.com/kairos-io/
   version = "master"
   url = "/docs"
 
+[[params.versions]]
+  version = "v3.2.4"
+  url = "/v3.2.4/docs"
+
```

Comment:
I'd remove this and update in the section below to only show 3.2.4 in the menu

Reply:
good point, fixing